scripts/utils/trading_strategy_helpers.js

```diff
@@ -796,10 +848,21 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
    * @param {Withdrawal[]} withdrawals List of {@link Withdrawal} that are to be bundled together
    * @returns {Transaction} Multisend transaction that has to be sent from the master address to transfer back the fubnds stored in the exchange
```

Comment:
Since we are at it:
```suggestion
   * @returns {Transaction} Multisend transaction that has to be sent from the master address to transfer back the funds stored in the exchange
```